### ISSUE TITLE
int has no such attribute: __and__

### ISSUE THREAD

**Issue by plombardi89** [datawire/quark#106]
Looks like you guys don't cast to bool?

```java
      @doc("Indicates whether a route exists within the routing table")
      bool hasRoute(String name) {
        return data.contains(name) && (data[name].size() > 0);
      }
```

**Comment by plombardi89** [comment]
Found the issue... I think

```java
macro int contains(K key) $java{($self).containsKey($key)}
                                  $py{($key) in ($self)}
                                  $js{($self).has($key)};
```

**Comment by bozzzzo** [comment]
Fixed in f1410e08a2c
